src/controller/TypedReadCallback.h

```diff
@@ -153,6 +154,26 @@ class TypedReadEventCallback final : public app::ReadClient::Callback
 
     void OnError(const app::ReadClient * apReadClient, CHIP_ERROR aError) override { mOnError(nullptr, aError); }
 
+    void OnDeallocatePaths(chip::app::ReadPrepareParams && aReadPrepareParams) override
+    {
+        if (aReadPrepareParams.mpAttributePathParamsList != nullptr)
```

Comment:
This is the _event_ callback.  Where would these attribute paths come from?

And this PR missed adding OnDeallocatePaths to the read callback here so that one leaks.

Perhaps OnDeallocatePaths should not have a default "just leak" impl?   At the very least that impl seems like it should abort....

Reply:
Updated. would do follow-up to do the right abort if not implemented for OnDeallocatePaths thanks